docs/excel/make-custom-functions-compatible-with-xll-udf.md

```diff
@@ -44,15 +41,15 @@ The following example shows how to specify both a COM add-in and an XLL as equiv
 ```
 
 > [!NOTE]
-> If an add-in declares its custom functions to be XLL compatible, changing the manifest at a later time could break a user's workbook because it will change the file format.
+> If an Excel JavaScript API add-in declares its custom functions to be compatible with an XLL add-in, changing the manifest at a later time could break a user's workbook because it will change the file format.
 
 ## Custom function behavior for XLL compatible functions
 
-An add-in's XLL functions are converted to XLL compatible custom functions when a spreadsheet is opened and there is an equivalent add-in available. On the next save, the XLL functions are written to the file in a compatible mode so that they work with both the XLL and Excel add-in custom functions (when on other platforms).
+An add-in's XLL functions are converted to XLL compatible custom functions when a spreadsheet is opened and there is an equivalent add-in available. On the next save, the XLL functions are written to the file in a compatible mode so that they work with both the XLL add-in and Excel JavaScript API add-in custom functions (when on other platforms).
```

Comment:
What is this conversion all about? Are the XLL custom functions converted to JavaScript custom functions?